test/document_test.rb

```diff
@@ -30,7 +30,7 @@ def add_to_foo key, value
 end
 
 class WithTime < Dolly::Document
-  property :created_at
+  property :created_at, default: -> {DateTime.now}
```

Comment:
why not `Time` ?
http://stackoverflow.com/questions/1261329/whats-the-difference-between-datetime-and-time-in-ruby/21075654#21075654